assets/src/blocks/Cookies/useCookie.js

```diff
@@ -0,0 +1,47 @@
+import { useState, useEffect } from 'react';
+
+export const readCookie = (name) => {
+  const declarations = document.cookie.split(';');
+  let match = null;
+  declarations.forEach(part => {
+    const [key, value] = part.split('=');
+    if (key.trim() === name) {
+      match = value;
+    }
+  });
+  return match;
+};
+
+export const writeCookie = (name, value, days = 365) => {
```

Comment:
I made days optional because currently all calls to `writeCookie` have the same days, except for when the cookie is removed, which has a wrapper function around it.